pages/linux/pacman.md

```diff
@@ -3,7 +3,7 @@
 > Arch Linux package manager utility.
 > See also: `pacman-database`, `pacman-deptest`, `pacman-files`, `pacman-key`, `pacman-mirrors`, `pacman-query`, `pacman-remove`, `pacman-sync`, `pacman-upgrade`.
 > For equivalent commands in other package managers, see <https://wiki.archlinux.org/title/Pacman/Rosetta>.
-> More information: <https://man.archlinux.org/man/pacman.8>.
+> More information: <https://manned.org/pacman.8>.
```

Comment:
According to the current style guide, linking to the author's provided man page is preferred.
> On the `More information` link line, we prefer linking to the author's provided documentation of the command-line reference ***or the man page***.